sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java

```diff
@@ -315,7 +388,7 @@ private static void extendEnv(Protos.CommandInfo.Builder builder, Map<String, St
         return builder.build();
     }
 
-    private static void setHealthCheck(
+    private void setHealthCheck(
```

Comment:
(cotd from subjective comment in constructor) For example this (and similar calls elsewhere) could continue being static if useDefaultExecutor was being passed in.